build.gradle

```diff
@@ -202,12 +202,15 @@ project('spring-rabbit') {
 		compile project(":spring-amqp")
 
 		compile "com.rabbitmq:amqp-client:$rabbitmqVersion"
-		compile ("com.rabbitmq:http-client:$rabbitmqHttpClientVersion", optional)
+		compile ("com.rabbitmq:http-client:$rabbitmqHttpClientVersion") {
+			exclude group: 'org.springframework'
+		}
```

Comment:
OK. I see your comment:
> http-client is no longer optional - needed by the LQCF 

But that doesn't matter that we can't live with `optional`. Who wants to use `LQCF`, that must add that dep for his purposes.
WDYT?